crates/diffbot_lib/src/github/github_api.rs

```diff
@@ -244,34 +240,57 @@ pub async fn download_url<S: AsRef<str>>(
 ) -> Result<Vec<u8>> {
     let target = find_content(installation, repo, filename, commit).await?;
 
-    let download_url = target
-        .download_url
-        .as_ref()
-        .ok_or_else(|| format_err!("No download URL given by GitHub"))?;
-
-    let response = reqwest::get(download_url).await?;
+    let content = target.content
+        .ok_or_else(|| format_err!("File had no content!"))?
+        .replace("\n", "");
 
-    Ok(response.bytes().await?.to_vec())
+    general_purpose::STANDARD
+        .decode(content)
+        .map_err(|decode_error| format_err!("DecodeError: {}", decode_error))
 }
 
-pub async fn download_file<S: AsRef<str>>(
-    installation: &InstallationId,
-    repo: &Repository,
-    filename: S,
-    commit: S,
-) -> Result<PathBuf> {
-    let target = find_content(installation, repo, &filename, &commit).await?;
-
-    let mut path = PathBuf::new();
-    path.push(".");
-    path.push(DOWNLOAD_DIR);
-    path.push(&target.sha);
-    path.set_extension("dmi"); // Method should have an IDB qualifier due to being a shared crate
-
-    async_fs::create_dir_all(path.parent().unwrap()).await?;
-    let mut file = File::create(&path).await?;
-
-    let data = download_url(installation, repo, &filename, &commit).await?;
-    file.write_all(&data).await?;
-    Ok(path)
+/* local test requires commenting out the .installation(...) call in find_content(), a valid github token with access, and the following dep: actix-rt = "2.9.0"
```

Comment:
Should we remove this?

Reply:
nah, it could be used for reference for testing